.github/workflows/release.yml

```
@@ -0,0 +1,36 @@
+name: Release
+
+on: 
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with: { fetch-depth: 0 }
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
```

```suggestion
        uses: actions/setup-go@v5
```
Old NodeJS runtimes will stop working, so it's worth to use a newer one

---

fixed